db/restore.sh

```diff
@@ -1,4 +1,9 @@
 #!/bin/sh
+#
+#Copyright (c) 2018.
+#This file is part of globocom/zabbix-scripts
+#(see https://github.com/globocom/zabbix-scripts).
+#License: 3-clause BSD, see https://opensource.org/licenses/BSD-3-Clause
 
```

Comment:
Please add extra whitespace between # and the text. You can use the example above.

Reply:
Ok. Done! :+1: